specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -315,6 +315,7 @@ model AnalyzeDocumentRequest {
   Base64 encoding of the document to analyze.  Either urlSource or base64Source
   must be specified.
   """)
+  @projectedName("python", "bytesSource")
```

Comment:
The client library renames should only be done in client.tsp. This page has an example of how these decorators are specified in client.tsp: https://azure.github.io/typespec-azure/docs/howtos/DataPlane%20Generation%20-%20DPG/04renaming#model-names

Reply:
Thanks @catalinaperalta , updated!